utils/node.go

```diff
@@ -27,3 +27,12 @@ func QueryTxs(node node.Node, query string) ([]*coretypes.ResultTx, error) {
 
 	return txs, nil
 }
+
+// QueryBlock queries block details from the given node
+func QueryBlock(node node.Node, blockHeight int64) (*coretypes.ResultBlock, error) {
```

Comment:
Is this method really necessary?

Reply:
Yes this method helps to get the block details for the `feegrant` fix command. We also reuse it in PR #248 to get the block results for `block` fix command

Reply:
This can be deleted now since it's no longer used, right?